sections/data_release_production_pipelines.tex

```diff
@@ -71,26 +75,332 @@ \section{Data Release Production}
+    parallel for ccd in SCIENCE_SENSORS:
+        sources[ccd] = %\hr{ApplyApCorr}%(sources[ccd], calexp.psf)
+    return calexp, sources
+\end{lstlisting}
+
+\paragraph{Input Data Product: Raw}
+\label{sec:drpBootstrapImChar_raw}
+
+Raw amplifier images from science and wavefront CCDs, spread across one or more snaps.  Needed telescope telemetry (seeing estimate, approximate pointing) is assumed to be included in the raw image metadata.
+
+\paragraph{Input Data Product: Reference}
+\label{sec:drpBootstrapImChar_reference}
+
+A full-sky catalog of reference stars derived from both external (e.g. Gaia) and LSST data.
+
+The \hyperref[sec:drpStandardJointCal]{StandardJointCal} pipeline will later define a deeper reference catalog derived from this one and the new data being processed, but the origin and depth of the initial reference catalog is largely TBD.  It will almost certainly include Gaia stars, but it may also include data from other telescopes, LSST special programs, LSST commissioning observations, and/or the last LSST data release.
+
+\paragraph{Output Data Product: Source}
+\label{sec:drpBootstrapImChar_sources}
+
+A preliminary version of the Source table.  This could contain all of the columns in the DPDD Source schema if the \hr{MeasureSources} is appropriately configured, but some of these columns are likely unnecessary in its role as an intermediate data product that feeds \hyperref[sec:drpStandardJointCal]{StandardJointCal}, and it is likely that other non-DPDD columns will be present for that role.
+
+BootstrapImChar also has the capability to produce even earlier versions of the Source table for diagnostic purposes (see \hr{WriteDiagnostics}).  These tables are not associated with any photometric calibration or aperture correction, and some may not have any measurements besides centroids, and hence are never substitutable for the final Source table.
+
+\paragraph{Output Data Product: CalExp}
+\label{sec:drpBootstrapImChar_calexp}
+
+A preliminary version of the CalExp (calibrated direct exposure).  CalExp is an \hyperref[sec:spImagesExposure]{Exposure} object, and hence it has several components.  BootstrapImChar is the only pipeline that actually updates all of them.  Some CalExp components are determined at the scale of a full FoV and hence should probably be persisted at the visit level (PSF, WCS, Calib, Background), while others are straightforward CCD-level data products (Image, Mask, Variance).
+
+\paragraph{RunISR}
+\label{sec:drpBootstrapImChar_RunISR}
+
+Delegate to the \hyperref[sec:acISR]{ISR algorithmic component} to perform standard detrending as well as brighter-fatter correction and interpolation for pixel-area variations (\hyperref[sec:acFixPixelAreaVariations]{Warping Irregularly-Sampled Images}).  It is possible that these corrections will require a PSF model, and hence must be backed-out and recorrected at a later stage when an improved PSF model is available.
```

Comment:
ISR requires additional inputs, of course, which we don't call out as inputs to BoostrapImChar. That makes sense: we're not feeding these inputs through the pipeline in the same way as the data, and we don't want to tightly couple this section of the docs to the details of the calibration products/ISR procedure. It would be useful to call it out in some way, though: ultimately, we'll want to be able to quickly verify all the inputs and outputs of everything without digging down to Algorithmic Components.